### DISCUSSION TITLE
More Sausage/Meat Preservation Recipes

### DISCUSSION THREAD

**Discussion by PsyXypher** [cataclysmbn/Cataclysm-BN#2057] [General]
# Is your feature request related to a problem? Please describe.
So, I was butchering an a Sewer Gator and seeing what I could do with its meat. Since I didn't have enough fridge room to store all of it, I started looking into recipes I could use. I found the Wasteland Sausage (which will be the subject of another request) and found a serious problem: The wasteland sausage recipe requires a stomach to make, and the gator had only one stomach. 

This left 8 pieces of raw lung, 5 pieces of raw kidney, 11 brains, 22 raw liver and 5 sweetbreads. Lots of delicious meat that might be able to go to other recipes put to waste.

# Describe the solution you'd like
I believe there should be more ways to turn organ drops into food. While all of the food could be turned into "cooked" variations, there's not much reason to do so. Most of them have requirements that are unfeasible or require perishable/extensive farming (such as onions). There should be a way to preserve meat in other forms. I can't imagine grinding up a bunch of raw meat would be super appetizing but it'd be more efficient. 

# Describe alternatives you've considered
Stomach drops could be reworked to be based on the animal size. Something akin to "Stomach portion". This would allow for making wasteland sausage based on animal size; it's a little ridiculous you could turn an animal like an alligator into a single sausage.

**Comment by Relgar** [comment]
May be worth examining how all meat products should be treated as compared to chunks of meat (which can be canned two ways, vacuum sealed, dehydrated, pickled, etc.). This could be a json change. However, this raises the question of why these need to be differentiated - should offal be a component something special that can only be made rarely in comparison with normal meat? What are the advantages / disadvantages of doing this?

**Comment by Pr0man** [comment]
This is basically an artefact of a bygone age.
Back in the glory days of mutant meat and the likes not existing, there used to be a generic "offal" item that more or less just summarized internal organs and stomach drops were a bunch more common, making the recipe for wasteland sausage a whole lot more affordable, as you generally got enough stomachs from medium-sized mobs to turn the generic offal drops into wasteland sausages to not waste them. A rebalance for stomach drops or a synthetic replacement as we use it in todays meat industry seems like the most sensible choice without relegating the recipe to obscurity. Collagen sausage skins for example are edible and made from animal hide produce.
